### PR TITLE
add code of conduct

### PR DESCRIPTION
**Pull request by berlintam**
closes https://github.com/rubycorns/rorganize.it/issues/366

### CODE REVIEW

**Comment by PragTob** [comment]
:+1: the from contributor convenant is also what we have over at shoes, so I like it :+1:

**Comment by berlintam** [comment]
plus there's a pic! :smile: :rubycorn:

**Comment by PragTob** [comment]
Also the coc is app level, I think we should add one to the github repository and mention it in the README much like in [shoes 4 land](https://github.com/shoes/shoes4/blob/master/CODE_OF_CONDUCT.md)

File: app/views/pages/conduct.html.haml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+    %h2 Background & Reporting
+    %p 
+      A Code of Conduct was recently added to 
+      %a{ href: "https://github.com/rails/rails/pull/21292" } Rails. 
+      As a project group, we were thrilled to see that this addition was met with numberous thumbs up from the community, and were inspired to add one ourselves. Our Code of Conduct is the same as theirs. 
+      %p 
+        To report an issue, please send us an 
+        %a{ href: "mailto:roar@rorganize.it" } email.
+        We are here to help and want ot make the rorganize community a wonderful and safe place.
```
Comment (sareg0) [inline app/views/pages/conduct.html.haml]: '...ot make the...' needs to be '... to make the...'

**Comment by sareg0** [comment]
the picture is awesome!! CoC is awesome! I made a comment re. a super minor typo.

**Comment by berlintam** [comment]
thanks y'all! I added/fixed all the things.

**Comment by til** [comment]
:+1:

Also I just pushed a commit to this PR that wraps all the long text
lines. I hope nobody objects. In my opinion, especially for markdown, it
is nicer this way.

File: README.md [hunk 2]
```diff
@@ -13,6 +13,10 @@ We've had lots of people asking us if this project is open for all to work on an
 even though they are a) not women, b) not in our project group, c) don't know us, d) some other reason. The answer is of course yes! We are not a closed group and we welcome your PRs. All we ask is that you communicate with us regularly, make clean PRs, test your work, and be nice. If you write some crazy code, explanations are helpful so that we can learn
 from what you've done and become better coders ourselves in the process. If you have a great idea for a feature & want to implement it, please open an issue, assign yourself to it, and hack away :)
 
+### Code of Conduct
+
+You can learn about our Code of Conduct [here](http://rorganize.it/conduct) and [here](https://github.com/rubycorns/rorganize.it/blob/master/CODE_OF_CONDUCT.md)
+
```
Comment (til) [inline README.md]: This produces two links that both have the link text "here". For accessibility reasons such as screen readers, but also in general, I think it is better practice to use some meaningful text as link text. I'm not 100% what the difference between the two versions is, so maybe my suggestion is off, please improve if you have a better one:

```markdown
Here you can find our [Code of Conduct](https://github.com/rubycorns/rorganize.it/blob/master/CODE_OF_CONDUCT.md)
and here [Code of Conduct with Background](http://rorganize.it/conduct).
```

File: app/views/pages/about.html.haml [hunk 3]
```diff
@@ -24,3 +24,7 @@
     %p
       %a{ href: "https://shellycloud.com/", target: "_blank"}
         %img{ src: "//shellycloud.com/assets/badge.png", alt: "Shellycloud badge"}
+    %h1.page-header Code of Conduct
+    %p
+      Read about our Code of Conduct
+      %a{ href: conduct_path }  here
```
Comment (til) [inline app/views/pages/about.html.haml]: Same remark re link text like above.

File: README.md (hunk 2 above)
Comment (berlintam) [inline README.md]: good point @til!

File: app/views/pages/conduct.html.haml [hunk 4]
```diff
@@ -0,0 +1,78 @@
+.container
+  .row
+
+    %h2 Background & Reporting
+    %p 
+      A Code of Conduct was recently added to 
+      %a{ href: "https://github.com/rails/rails/pull/21292" } Rails. 
+      As a project group, we were thrilled to see that this addition
+      was met with numberous thumbs up from the community, and were
```
Comment (til) [inline app/views/pages/conduct.html.haml]: Should be "numerous"

**Comment by berlintam** [comment]
:shipit: ?

**Comment by berlintam** [comment]
cat

**Comment by sareg0** [comment]
Thank you for putting this in!!

On Tuesday, 25 August 2015, Tam <notifications@github.com> wrote:

> Merged #367 <https://github.com/rubycorns/rorganize.it/pull/367>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/rubycorns/rorganize.it/pull/367#event-391930823>.
>


-- 
Kindest Regards,
Sara and the Rails Girls Summer of Code Team
@RailsGirlsSoC <https://twitter.com/RailsGirlsSoC>

Kindest Regards,
Sara
@sareg0 <https://twitter.com/sareg0>
+49(0)17699393251

**Pull request merged**